tasks.py

```diff
@@ -57,6 +57,22 @@ def server(host=None, port=5000, debug=True, live=False):
 
 
 @task
+def debug_server(host=None, port=5000, debug=True, live=False):
```

Comment:
Can you please explain the use case for this task? Typically we just add:
`import ipdb; ipdb.set_trace()` to trigger a breakpoint in the code.